src/components/swap/SwapModalFooter.tsx

```diff
@@ -117,11 +117,12 @@ export default function SwapModalFooter({
           error={severity > 2}
           style={{ margin: '10px 0 0 0' }}
           id="confirm-swap-or-send"
-        >
-          <Text fontSize={13} fontWeight={600}>
-            {severity > 2 ? 'Swap Anyway' : 'Confirm Swap'}
-          </Text>
-        </ButtonError>
+          content={
+            <Text fontSize={13} fontWeight={600}>
+              {severity > 2 ? 'Swap Anyway' : 'Confirm Swap'}
```

Comment:
I know it's not your fault but can we start changing these magic numbers but the actual severity?
I think this one in particular is MEDIUM.

Reply:
100% this shouldn't be hardcoded!

Reply:
will make issue for it!